analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/MetricsAnalyzerTest.cs

```diff
@@ -61,24 +61,24 @@ public void VerifyMetrics_Razor()
         {
             using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
 
-            CreateBuilder(false, RazorFileName)
+            CreateBuilder(false, RazorFileName, "Component.razor")
                 .WithSonarProjectConfigPath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
                 .VerifyUtilityAnalyzer<MetricsInfo>(messages =>
                 {
-                    messages.Should().HaveCount(2); // Razor.razor and _Imports.razor
+                    messages.Should().HaveCount(3); // Razor.razor, Component.razor and _Imports.razor
```

Comment:
We could change this assertion to check the file names as we did in the other tests.